platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/ClassPageRenderer.java

```diff
@@ -113,9 +113,29 @@ protected void renderFailures(SimpleHtmlWriter htmlWriter) throws IOException {
         }
     }
 
+    @Override
+    protected void renderIgnoredTests(SimpleHtmlWriter htmlWriter) throws IOException {
```

Comment:
Do you have a screenshot of this?

Reply:
<img width="942" alt="image" src="https://github.com/user-attachments/assets/9398a106-0163-4af4-b773-fc17fdd376c5" />

Reply:
What if there are failing tests and ignored tests?

Which tab comes first?

Which tab is selected first?

I think the answer to both of those questions should be the failing tests.

Reply:
I strongy agree with Justin here, failing is more important.

Reply:
Failed would come first (to the left) if both are shown.

The left most tab is the default one.  That means if you only have skipped tests (and no failures), that's what's shown by default.  That might be weird, but this is how it's worked on other pages for awhile.